Alexey/Text/vigenere_cipher/tests/test_vigenere_cipher.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+from Alexey.Text.vigenere_cipher.main.vigenere_cipher import generate_key, original_text, cipher_text_func
+
+
+class TestVigenereCipher(unittest.TestCase):
+    def test_send_word_and_key_and_expect_generated_key(self):
+        expect_set = 'SEASEASEAS'
+        actual_set = generate_key('bigbrother', 'SEA')
```

Comment:
Add test when condition` if len(word) == len(key):` will be true
General rule - when you have if statement it automatically add one more test. You should check when IF is true and when IF is false . Here is only 'False' condition